tools/gulp/constants.ts

```diff
@@ -22,6 +24,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+export const LICENSE_BANNER = `/**
+  * @license Angular Material v${MATERIAL_VERSION}
+  * Copyright (c) 2016 Google, Inc. https://angular.io/
```

Comment:
Don't think that would play a big role. Just switched it to `material.angular.io`